src/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncEngine.Unix.cs

```diff
@@ -46,16 +46,29 @@ public bool TryRegister(SafeCloseSocket socket, Interop.Sys.SocketEvents current
             }
         }
 
-        private const int EventBufferCount = 64;
+        private const int EventBufferCount =
```

Comment:
There are some cases like this in the implementation already. This makes it easier for test to hit the boundary conditions. I am not sure if it applies here too.